restfulpy/orm/mixins.py

```diff
@@ -50,8 +50,11 @@ def last_modification_time(self):
 
     @staticmethod
     def before_update(mapper, connection, target):
-        if target.object.__exclude__.issubset(target.unmodified):
-            target.object.modified_at = datetime.utcnow()
+        exclude = target.object.__exclude__ or set()
```

Comment:
Use 
```python
class Mixin:
    __exclude__ = set()
```
 Instead of instantiating a set on every request.

Reply:
Thank you for this point.